src/components/AdoptionProcess/AdoptionProcess.css

```diff
@@ -0,0 +1,23 @@
+.container {
+    text-align: center;
+}
+
+.adoption-container img {
```

Comment:
className are camelCase

Reply:
Hi Halit, I changed the naming of classes and pushed my changes